main.go

```diff
@@ -29,17 +30,46 @@ func NewJoke(id, title, body string, score int) Joke {
 	return Joke{id, title, body, score}
 }
 
+// Page struct
+
+type Page struct {
+	Skip    int
+	Seed    int
+	Content []Joke
+}
+
+//GLOBAL VARIABLES
+
 var jokes []Joke
 
+var t *template.Template
+
 // HANDLERS
 
 func getJokes(w http.ResponseWriter, r *http.Request) {
-	t, err := template.ParseFiles("templates/index.html", "templates/header.html", "templates/footer.html")
+	var page Page
+
+	skip, err := strconv.Atoi(r.URL.Query().Get("skip"))
+	if err != nil {
+		fmt.Println("Converting err, using default value")
```

Comment:
you can use default value if the was know specified any parameter, but if there is something bad in it, it's better to create a response with http status 400.